src/components/MenuPanel/OrganizationSwitcher/Favorites.js

```diff
@@ -63,6 +64,11 @@ class Favorites extends React.Component {
     }
   }
 
+  handleGoHome = () => {
+    window.location.hash = ''
+    this.props.onDone()
```

Comment:
Maybe `onDone()` could be renamed to `onUpdate()`?

Also, isn't this behaviour a bit odd? I think we clear the `StoredList` of favourited organizations every time we call `this.props.onDone()` without any arguments.

Reply:
Yes you’re right, it’s only working because `onDone` gets called on unmount, with the right data, right after… the empty `onDone()` call was probably from a previous iteration 😓

And I also agree about `onDone` => `onUpdate`.